docs/src/pages/components/data-grid/rows/InfiniteLoadingGrid.js

```diff
@@ -14,23 +14,23 @@ function CustomLoadingOverlay() {
 }
 
 export default function InfiniteLoadingGrid() {
-  const [size, setSize] = React.useState(20);
+  const [rowLengthSize, setRowLengthSize] = React.useState(20);
```

Comment:
I don't understand why you need this state

Reply:
I can explain - this was needed when the `virtualRowsCount` were not provided in the params but since they are it isn't needed anymore (assuming we will keep the params as they are now). Otherwise, you need to keep the state.

Reply:
updated